### PR TITLE
Reorganizing sample project for better documentation use

### PR DESCRIPTION
**Pull request by gled4er**
Hello @cgillum ,

May I ask you to review this pull request with small changes to the samples project for better references in the documentation?

Thank you!

### CODE REVIEW

File: samples/VSSample.Tests/HelloSequenceActivityTests.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using Xunit;
```
Comment (cgillum) [inline samples/VSSample.Tests/HelloSequenceActivityTests.cs]: Please be sure to include the copyright header on all new files.

File: samples/VSSample.Tests/HttpStartTests.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
```
Comment (cgillum) [inline samples/VSSample.Tests/HttpStartTests.cs]: Can any of the above using statements be removed? This seems like a lot.

File: samples/VSSample.Tests/HelloSequenceActivityTests.cs (hunk 1 above)
Comment (gled4er) [reply samples/VSSample.Tests/HelloSequenceActivityTests.cs]: Yes, of course. Thank you!

**Comment by gled4er** [comment]
Hello @cgillum ,

I implemented the changes you requested.

Please let me know what should be the next issues I should fix.

Thank you!

**Pull request merged**